scripts/data_collection/update_dataset.py

```diff
@@ -0,0 +1,83 @@
+"""
+This script allows the user to clone repositories listed in dataset from GitHub
+or pull changes from repository, if it is already stored.
+Each repository is cloned without history (or with, if necessary and argument `store-history` is passed).
+When the argument `save-to-db` is passed,
+the date of the last pull of each repository is saved to the database configured by `database.ini` file
+Script accepts
+    * path to CSV file --  dataset downloaded from https://seart-ghs.si.usi.ch/
+    * path to the output directory, where repositories are cloned
+    * index to start from
+    * whether to use database or not
+"""
+import datetime
+import logging
+import pandas as pd
+import argparse
+import os
+
+from data_collection.db_connect import DatabaseConn
+from data_collection.repositories_table import RepositoriesTable
+from data_collection.git_repo import GitRepository
+from utils import create_directory
+
+logging.basicConfig(level=logging.DEBUG)
+
+
+def update_dataset(input_path: str, output_path: str, start_from: int, save_to_db: bool, store_history: bool):
```

Comment:
Maybe you create a class for this config? Too much the number of arguments now in the function